### DISCUSSION TITLE
discuss: Improve bug report template

### DISCUSSION THREAD

**Discussion by tzssangglass** [apache/apisix#4785] [General]
# Improve Docs

## Please describe which part of docs should be improved or typo fixed

Improve [bug report template](https://github.com/apache/apisix/blob/master/.github/ISSUE_TEMPLATE/bug.md)

The current bug report template is adequate, but it is in markdown format, and we still see issues that do not follow the formatting of the issue, making it look tired.

## Describe the solution you'd like

I saw a better bug report template, here: https://github.com/stackblitz/webcontainer-core/blob/main/.github/ISSUE_TEMPLATE/bug_report.yml

This one looks more interfaced
![image](https://user-images.githubusercontent.com/30819887/127959530-bbe9ead4-ac55-4f64-a21b-f62b05154d38.png)

I hope this will help reduce the number of poorly formatted issues.

**Comment by juzhiyuan** [comment]
agree +1

**Comment by Baoyuantop** [comment]
agree +1

**Comment by bzp2010** [comment]
agree +1

**Comment by tokers** [comment]
agree +1
